File: LP_Python/loop_looping/listas/loop_5.py
CountTotal = 0
CountFem = 0 
CountMan = 0
CountKid = 0
CountFemL = 0
nLivros = 0
CountIdade = 0
CountMenNL = 0

while True:
    sexo = str(input("Digite o sexo do participante: "))
    idade = int(input("Digite a idade do participante: "))
    livros = int(input("Digite a quantidade de livros lidos em 2006: "))
    if idade<0:
        break
    
    CountTotal+=1
    
    if sexo=="f":
        CountFem+= 1

    elif sexo=="m":
        CountMan+=1

    if livros==0:
        nLivros+=1

    if idade>10:
        CountKid+=1

    if sexo=="f" and livros>=5:
        CountFemL+=1

    if sexo=="m" and livros<5:
        CountIdade+=idade
        CountMenNL+=1

MDmenl = CountIdade/CountMenNL
PCnl = (nLivros/CountTotal) * 100

print("qnt de livros lidos por crianças menores de 10 anos: %d\nnúmero de mulheres que leram 5 livros ou mais: %d\nmedia da idade dos homens que leram menos que 5 livros: %.2f\npercentual de pessoas que nao leram nenhum livro %.2f%%\n "%(CountKid,CountFemL,MDmenl,PCnl))

File: LP_C++/lista_loop/loop_5.cpp
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
	int idade=0,Qlivros=0,Qlivros10=0,Qmulheres5=0,Qentevis=0,Qnlivros=0,mdIdade=0,Qmen=0,Qmen5=0;
	char sexo;
	
	do
	{
		printf("\nDigite o sexo do entrevistado: ");
		scanf("%s",&sexo);
		printf("\nDigite a idade do entrevistado: ");
		scanf("%d",&idade);
		printf("\nDigite a quantidade de livros lidos em 2006:");
		scanf("%d",&Qlivros);
		Qentevis++;
		mdIdade+=idade;
		if (idade<10)
		{
			Qlivros10+=Qlivros;
		}
		if (sexo=='m')
		{
			Qmen++;
		}		
		if ((sexo=='m')&&(Qlivros<5))
		{
			Qmen5++;
		}
		if ((sexo=='f')&&(Qlivros>=5))
		{
			Qmulheres5++;
		}
		if (Qlivros==0)
		{
			Qnlivros++;
		}
		
	} while (idade>=0);
	float Pcnlivros=(float)(Qnlivros/Qentevis)*100;
	float MDmen5=(float)Qmen5/Qmen;
	printf("\nQuantidade total de livros com menos de 10 anos: %d",Qlivros10);
	printf("\nQuatidades de mulheres que leram 5 livros ou mais: %d",Qmulheres5);
	printf("\nMédia de idade dos homens que leram 5 livros ou mais: %.2f",MDmen5);
	printf("\nPercentual de pessoas que não leram livros: %.2f",Pcnlivros);


	return 0;
}
